src/components/forms/event-form.js

```diff
@@ -1838,15 +1838,14 @@ class EventForm extends React.Component {
                                   score.ranking_type_id
                                 )}
                                 data-tooltip-id="help"
+                                data-tooltip-variant="light"
```

Comment:
- why u use data-tooltip-variant="light" ?
- on the doc ( https://react-tooltip.com/docs/options#available-attributes) u see there is a variant prop already